### DISCUSSION TITLE
"chia farm summary" output on the main node with multiple farmers is useless

### DISCUSSION THREAD

**Discussion by happycouak** [Chia-Network/chia-blockchain#6467] [Q&A] [answered]
Apologies if the question as been already asked but didnt found any answers yet.

Following the doc here [1], I have just migrated my setup from one node without any local plots (full_node + farmer + harvester) connected to different NAS which were sharing plots overs the network to the main, to harvester installed on each NAS forwarding signages/proofs to the main node.

So far so good.

Thing now is the ouput of "chia farm summary" on the main node, which I were monitoring with care before, didn't show anymore all this useful information like plots count, total size of plots and expected time to win:

```
Farming status: Farming
Total chia farmed: 0.0 <===
User transaction fees: 0.0
Block rewards: 0.0 <===
Last height farmed: 0 <===
Plot count: 0 <===
Total size of plots: 0.000 GiB <===
Estimated network space: 18664.575 PiB
Expected time to win: Unknown <===
Note: log into your key using 'chia wallet show' to see rewards for each key
```

Is it the expected behavior ? is there a way to get back these useful informations on the main node ?

[1] https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines

**Comment by 21497724** [comment]
Same issue.

**Comment by gades** [comment]
Also, the useless information from slave node , any way to add Farming status:  and other parameters ?
```
Connection error. Check if wallet is running at 9256
Connection error. Check if full node is running at 8555
Connection error. Check if farmer is running at 8559
Farming status: Not available
Total chia farmed: Unknown
User transaction fees: Unknown
Block rewards: Unknown
Last height farmed: Unknown
Plot count: 17
Total size of plots: 1.683 TiB
Estimated network space: Unknown
Expected time to win: Unknown
Note: log into your key using 'chia wallet show' to see rewards for each key
```

**Comment by hengic** [comment]
I have the same issue. Is it affecting the reward of harvesting or only a display issue?

**Reply by firmicus** [reply]
I'm in the same situation, only have a few plots on the farmer but 50TB on the remote harvester.
Is it affecting my potential reward?

Many thanks'

**Comment by rmica** [comment]
It's a display issue. I solved it with ChiaMonitor. https://github.com/ytx1991/ChiaMonitor

**Reply by gades** [reply]
But it's third party software and we're expecting it in the core app because it's showing wrong information and it's not clear for understanding process of mining

**Reply by gades** [reply]
Also, we can use zabbix with UserParrameter

**Reply by rmica** [reply]
You're right but meanwhile it can help. I like the zabbix way.

**Reply by gades** [reply]
https://github.com/gades/zabbix-chia-harvester here is first template for monitoring

**Comment by hengic** [comment]
@rmica Thank you very much. It's quite helpful. But there're some fields not shown properly. See below pic.
![Uploading Screen Shot 2021-06-09 at 14.43.27.png…]()

**Reply by w4d3j** [reply]
I don't see any  pic...

**Comment by hengic** [comment]
<img width="1201" alt="Screen Shot 2021-06-09 at 8 02 25 PM" src="https://user-images.githubusercontent.com/22890997/121350776-a3ab8300-c95d-11eb-9240-7fcb902fa3d5.png">

Many fields show "No Data"

**Reply by rmica** [reply]
Check the paths in ChiaMonitor/data_collect/run_mtail.sh (or .bat), on the farmer node. Fix the default "../venv/bin/chia farm summary > farmer_summary.log" to your chia venv path.

**Reply by hengic** [reply]
thanks. it's the path issue.

**Comment by loppefaaret** [comment]
it is just a display issue - and is currently planned to be expanded on with an updated farmer <-> harvester protocol - but it is not top priority

**Comment by gades** [comment]
Added template for monitoring chia harvest nodes 

https://github.com/gades/zabbix-chia-harvester

<img width="1587" alt="Screenshot 2021-06-09 at 7 49 10 PM" src="https://user-images.githubusercontent.com/4200730/121396204-cdfc4100-c95b-11eb-9c40-7998d8fe6566.png">

**Reply by gades** [reply]
<img width="1594" alt="Screenshot 2021-06-10 at 5 43 55 PM" src="https://user-images.githubusercontent.com/4200730/121545755-78836b00-ca13-11eb-815b-8bd7d04749cb.png">

Added total count of plots

**Reply by j4r3kb** [reply]
Only way to monitor a harvester is to run `tail -f ~/.chia/mainnet/log/debug.log` with debug level INFO in config.

**Reply by rmica** [reply]
ChiaMonitor does it.

**Comment by emlowe** [comment, marked as answer]
We believe this is resolved in 1.2

**Comment by happycouak** [comment]
Solved by v 1.2.

**Reply by gades** [reply]
yes, but they did broke information about count of local plots on remote harvester 

`chia farm summary 
Connection error. Check if farmer is running at 8559
Connection error. Check if full node is running at 8555
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f1bcb83c160>
Connection error. Check if farmer is running at 8559
Farming status: Not available
Plot count: Unknown
Total size of plots: Unknown
Estimated network space: Unknown
Expected time to win: Unknown
For details on farmed rewards and fees you should run 'chia start wallet' and 'chia wallet show'`
